libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx

```diff
@@ -292,91 +293,181 @@ export function Workspace () {
     )
   }
 
+  const workspaceMenuIcons = [
+    <OverlayTrigger
+      placement='right-start'
+      overlay={
+        <Tooltip id="workspaceRenametooltip">
+          <span>Rename Workspace</span>
+        </Tooltip>
+      }
+    >
+      <span
+        hidden={currentWorkspace === LOCALHOST || currentWorkspace === NO_WORKSPACE}
+        id='workspaceRename'
+        data-id='workspaceRename'
+        onClick={(e) => {
+          e.stopPropagation()
+          renameCurrentWorkspace()
+          _paq.push(['trackEvent', 'fileExplorer', 'workspaceMenu', 'workspaceRename'])
+          hideIconsMenu(!showIconsMenu)
+        }}
+        className='far fa-edit w-100'>
+          {'  Rename'}
```

Comment:
IMHO it's better to style with css, For that please separate icon and the label into 2 separate spans and fix the spacing between them.

Reply:
Done. Used pl-2 rule for separation